srcpkgs/ironbar/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'ironbar'
 pkgname=ironbar
-version=0.14.1
+version=0.15.0
 revision=1
 build_style=cargo
 hostmakedepends="pkg-config"
-makedepends="gtk+3-devel gtk-layer-shell-devel"
+makedepends="gtk+3-devel gtk-layer-shell-devel libpulseaudio LuaJIT-devel"
+depends="lua54-lgi"
```

Comment:
Isn't this only required when custom Cairo widgets are configured? If so, it should be omitted from the dependency list.

Reply:
That's correct. The `cairo` feature is enabled by default but disabling it will remove the dep on luajit/lgi